Jenkinsfile

```diff
@@ -33,12 +33,12 @@ def getWorkspace = { buildType ->
 }
 
 @NonCPS
-def post_stage_failure(job, stage, error, url){
-  slackSend channel:"eax-builds", color: "danger", message: "Build ${job} failed in stage ${stage} with ${error}, see: <${url}|results>"
+def post_stage_failure(job, stage, build_id, url){
+  httpRequest url: "http://10.5.88.46:32768/hubot/jenkins-notify?room=eax-builds", httpMode: "POST", contentType: "APPLICATION_JSON", requestBody: "{\"build\": {\"phase\": \"FINISHED\", \"status\": \"FAILURE\", \"number\": ${build_id}, \"full_url\": \"${url}\"}, \"name\": \"${job} stage: ${stage}\" }"
```

Comment:
Eh unfortunately there doesn't seem to be any straight forward way to do it (Global properties plugin seems to be ignored if we use Jenkinsfile).